composer.json

```diff
@@ -20,8 +20,8 @@
         "php": "^7.4 || ^8.0"
     },
     "require-dev": {
-        "symfony/phpunit-bridge": "^5",
-        "phpstan/phpstan": "^1.3",
+        "phpunit/phpunit": "^9.5",
+        "phpstan/phpstan": "1.11.x-dev",
```

Comment:
since this package does not enforce a PHPStan beeing installed (and therefore also not a minimum version) - we need to feature detect whether the PHPStan version which is used, has all the required features